<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeMode</class>
 <widget class="QWidget" name="TimeMode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>484</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>484</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <property name="horizontalSpacing">
    <number>4</number>
   </property>
   <property name="verticalSpacing">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>320</width>
       <height>90</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>90</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
border-width: 3px;
border-color: rgb(148, 148, 148);
border-radius: 5px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QStackedWidget" name="stackedWidget_time">
     <property name="minimumSize">
      <size>
       <width>320</width>
       <height>363</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_3">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>2</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_from">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>from</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="3">
        <widget class="QComboBox" name="comboBox_first">
         <property name="minimumSize">
          <size>
           <width>260</width>
           <height>22</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Microseconds</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Milliseconds</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Seconds</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Minutes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Hours</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Days</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Weeks</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Years</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0" colspan="4">
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>320</width>
           <height>90</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>90</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">border-style: solid;
border-width: 3px;
border-color: rgb(148, 148, 148);
border-radius: 5px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_to">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>to</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="3">
        <widget class="QComboBox" name="comboBox_second">
         <property name="minimumSize">
          <size>
           <width>260</width>
           <height>22</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Microseconds</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Milliseconds</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Seconds</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Minutes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Hours</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Days</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Weeks</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Years</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="pushButton_Ct">
         <property name="minimumSize">
          <size>
           <width>105</width>
           <height>37</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1000</width>
           <height>1000</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>12</pointsize>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: rgb(220, 220, 220);
	font: 700 12pt &quot;Segoe UI&quot;;
	border-radius: 8px; 
}

QPushButton:hover {
	background-color: rgb(210, 210, 210);
	border-radius: 8px; 
}

QPushButton:pressed { 
	color: rgb(0, 0, 0);
 	border-width: 1px;
	border-radius: 10px;
 	font: bold 20px;
 	min-width: 8em;
 	padding: 5px;
}
</string>
         </property>
         <property name="text">
          <string>C</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QPushButton" name="pushButton_backspacet">
         <property name="minimumSize">
          <size>
           <width>105</width>
           <height>37</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1000</width>
           <height>1000</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>12</pointsize>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: rgb(220, 220, 220);
	font: 700 12pt &quot;Segoe UI&quot;;
	border-radius: 8px; 
}

QPushButton:hover {
	background-color: rgb(210, 210, 210);
	border-radius: 8px; 
}

QPushButton:pressed { 
	color: rgb(0, 0, 0);
 	border-width: 1px;
	border-radius: 10px;
 	font: bold 20px;
 	min-width: 8em;
 	padding: 5px;
}
</string>
         </property>
         <property name="text">
          <string>&lt;=</string>
         </property>
         <property name="shortcut">
          <string>Backspace</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QPushButton" name="pushButton_7t">
         <property name="minimumSize">
          <size>
           <width>105</width>
           <height>37</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1000</width>
           <height>1000</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>12</pointsize>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: rgb(234, 234, 234);
	font: 700 12pt &quot;Segoe UI&quot;;
	border-radius: 8px; 
}

QPushButton:hover {
	background-color: rgb(210, 210, 210);
	border-radius: 8px; 
}

QPushButton:pressed { 
	color: rgb(0, 0, 0);
 	border-width: 1px;
	border-radius: 10px;
 	font: bold 20px;
 	min-width: 8em;
 	padding: 5px;
}
</string>
         </property>
         <property name="text">
          <string>7</string>
         </property>
         <property name="shortcut">
          <string>7</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QPushButton" name="pushButton_8t">
         <property name="minimumSize">
          <size>
           <width>105</width>
           <height>37</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1000</width>
           <height>1000</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>12</pointsize>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: rgb(234, 234, 234);
	font: 700 12pt &quot;Segoe UI&quot;;
	border-radius: 8px; 
}

QPushButton:hover {
	background-color: rgb(210, 210, 210);
	border-radius: 8px; 
}

QPushButton:pressed { 
	color: rgb(0, 0, 0);
 	border-width: 1px;
	border-radius: 10px;
 	font: bold 20px;
 	min-width: 8em;
 	padding: 5px;
}
</string>
         </property>
         <property name="text">
          <string>8</string>
         </property>
         <property name="shortcut">
          <string>8</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QPushButton" name="pushButton_9t">
         <property name="minimumSize">
          <size>
           <width>105</width>
           <height>37</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1000</width>
           <height>1000</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>12</pointsize>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: rgb(234, 234, 234);
	font: 700 12pt &quot;Segoe UI&quot;;
	border-radius: 8px; 
}

QPushButton:hover {
	background-color: rgb(210, 210, 210);
	border-radius: 8px; 
}

QPushButton:pressed { 
	color: rgb(0, 0, 0);
 	border-width: 1px;
	border-radius: 10px;
 	font: bold 20px;
 	min-width: 8em;
 	padding: 5px;
}
</string>
         </property>
         <property name="text">
          <string>9</string>
         </property>
         <property name="shortcut">
          <string>9</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QPushButton" name="pushButton_4t">
         <property name="minimumSize">
          <size>
           <width>105</width>
           <height>37</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1000</width>
           <height>1000</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>12</pointsize>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: rgb(234, 234, 234);
	font: 700 12pt &quot;Segoe UI&quot;;
	border-radius: 8px; 
}

QPushButton:hover {
	background-color: rgb(210, 210, 210);
	border-radius: 8px; 
}

QPushButton:pressed { 
	color: rgb(0, 0, 0);
 	border-width: 1px;
	border-radius: 10px;
 	font: bold 20px;
 	min-width: 8em;
 	padding: 5px;
}
</string>
         </property>
         <property name="text">
          <string>4</string>
         </property>
         <property name="shortcut">
          <string>4</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QPushButton" name="pushButton_5t">
         <property name="minimumSize">
          <size>
           <width>105</width>
           <height>37</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1000</width>
           <height>1000</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>12</pointsize>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: rgb(235, 235, 235);
	font: 700 12pt &quot;Segoe UI&quot;;
	border-radius: 8px; 
}

QPushButton:hover {
	background-color: rgb(210, 210, 210);
	border-radius: 8px; 
}

QPushButton:pressed { 
	color: rgb(0, 0, 0);
 	border-width: 1px;
	border-radius: 10px;
 	font: bold 20px;
 	min-width: 8em;
 	padding: 5px;
}
</string>
         </property>
         <property name="text">
          <string>5</string>
         </property>
         <property name="shortcut">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item row="6" column="3">
        <widget class="QPushButton" name="pushButton_6t">
         <property name="minimumSize">
          <size>
           <width>105</width>
           <height>37</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1000</width>
           <height>1000</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>12</pointsize>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: rgb(234, 234, 234);
	font: 700 12pt &quot;Segoe UI&quot;;
	border-radius: 8px; 
}

QPushButton:hover {
	background-color: rgb(210, 210, 210);
	border-radius: 8px; 
}

QPushButton:pressed { 
	color: rgb(0, 0, 0);
 	border-width: 1px;
	border-radius: 10px;
 	font: bold 20px;
 	min-width: 8em;
 	padding: 5px;
}
</string>
         </property>
         <property name="text">
          <string>6</string>
         </property>
         <property name="shortcut">
          <string>6</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QPushButton" name="pushButton_1t">
         <property name="minimumSize">
          <size>
           <width>105</width>
           <height>37</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1000</width>
           <height>1000</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>12</pointsize>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: rgb(234, 234, 234);
	font: 700 12pt &quot;Segoe UI&quot;;
	border-radius: 8px; 
}

QPushButton:hover {
	background-color: rgb(210, 210, 210);
	border-radius: 8px; 
}

QPushButton:pressed { 
	color: rgb(0, 0, 0);
 	border-width: 1px;
	border-radius: 10px;
 	font: bold 20px;
 	min-width: 8em;
 	padding: 5px;
}
</string>
         </property>
         <property name="text">
          <string>1</string>
         </property>
         <property name="shortcut">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QPushButton" name="pushButton_2t">
         <property name="minimumSize">
          <size>
           <width>105</width>
           <height>37</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1000</width>
           <height>1000</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>12</pointsize>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: rgb(234, 234, 234);
	font: 700 12pt &quot;Segoe UI&quot;;
	border-radius: 8px; 
}

QPushButton:hover {
	background-color: rgb(210, 210, 210);
	border-radius: 8px; 
}

QPushButton:pressed { 
	color: rgb(0, 0, 0);
 	border-width: 1px;
	border-radius: 10px;
 	font: bold 20px;
 	min-width: 8em;
 	padding: 5px;
}
</string>
         </property>
         <property name="text">
          <string>2</string>
         </property>
         <property name="shortcut">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item row="7" column="3">
        <widget class="QPushButton" name="pushButton_3t">
         <property name="minimumSize">
          <size>
           <width>105</width>
           <height>37</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1000</width>
           <height>1000</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>12</pointsize>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: rgb(234, 234, 234);
	font: 700 12pt &quot;Segoe UI&quot;;
	border-radius: 8px; 
}

QPushButton:hover {
	background-color: rgb(210, 210, 210);
	border-radius: 8px; 
}

QPushButton:pressed { 
	color: rgb(0, 0, 0);
 	border-width: 1px;
	border-radius: 10px;
 	font: bold 20px;
 	min-width: 8em;
 	padding: 5px;
}
</string>
         </property>
         <property name="text">
          <string>3</string>
         </property>
         <property name="shortcut">
          <string>3</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="QPushButton" name="dot">
         <property name="minimumSize">
          <size>
           <width>105</width>
           <height>37</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1000</width>
           <height>1000</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>12</pointsize>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: rgb(234, 234, 234);
	font: 700 12pt &quot;Segoe UI&quot;;
	border-radius: 8px; 
}

QPushButton:hover {
	background-color: rgb(210, 210, 210);
	border-radius: 8px; 
}

QPushButton:pressed { 
	color: rgb(0, 0, 0);
 	border-width: 1px;
	border-radius: 10px;
 	font: bold 20px;
 	min-width: 8em;
 	padding: 5px;
}
</string>
         </property>
         <property name="text">
          <string>.</string>
         </property>
         <property name="shortcut">
          <string>.</string>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QPushButton" name="pushButton_0t">
         <property name="minimumSize">
          <size>
           <width>105</width>
           <height>37</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1000</width>
           <height>1000</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>12</pointsize>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: rgb(234, 234, 234);
	font: 700 12pt &quot;Segoe UI&quot;;
	border-radius: 8px; 
}

QPushButton:hover {
	background-color: rgb(210, 210, 210);
	border-radius: 8px; 
}

QPushButton:pressed { 
	color: rgb(0, 0, 0);
 	border-width: 1px;
	border-radius: 10px;
 	font: bold 20px;
 	min-width: 8em;
 	padding: 5px;
}
</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="shortcut">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="8" column="3">
        <widget class="QPushButton" name="answer">
         <property name="minimumSize">
          <size>
           <width>105</width>
           <height>37</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1000</width>
           <height>1000</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	font: 100 12pt &quot;Arial&quot;;
	border-radius: 8px; 
	background-color: rgb(157, 217, 255);
}

QPushButton:hover {
	background-color: rgb(105, 190, 255);
	border-radius: 8px; 
}

QPushButton:pressed { 
	color: rgb(0, 0, 0);
 	border-width: 1px;
	border-radius: 10px;
 	font: bold 16px;
 	min-width: 8em;
 	padding: 5px;
}
</string>
         </property>
         <property name="text">
          <string>answer</string>
         </property>
         <property name="shortcut">
          <string>Return</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
